templates/observables/br_d_l_to_d_l.cpp.in

```diff
@@ -0,0 +1,267 @@
+ */
+template <class Down>
+void write_btosmumu(
+   const Eigen::Array<std::complex<double>, 10, 1>& coeffs,
+   const @ModelName@_cxx_diagrams::context_base& context,
+   const softsusy::QedQcd& qedqcd) {
+
+   const auto complex_CKM = qedqcd.get_complex_ckm();
+
+   const std::complex<double> Vtb (complex_CKM(2, 2));
+   const std::complex<double> Vts (complex_CKM(2, 1));
+   const auto GF = qedqcd.displayFermiConstant();
+   const auto normalization = 4.*GF/Sqrt(2.) * Vtb*Conj(Vts) *
+      Sqr(unit_charge(context))*oneOver16PiSqr;
+
+   if (is_zero(std::abs(normalization))) {
+      ERROR("btosmumu: Normalization factor is vanishing.");
+      exit(EXIT_FAILURE);
+   }
+
+   const auto mb = context.mass<Down>({2});
+
+   // Dear reader, be aware of vector bosons (in loops) for b -> s mu mu.
+   // Some scheme dependence might be important.
+
+   // Compare to the basis in <DLDL/main.m>.
+   // They put SL on the first positions, we put BL.
+
+   // Here is the place for C7_bs = -1/2 A2R / normalization
+   // Here is the place for C7p_bs = -1/2 A2L / normalization
+
+   const auto C9_bsmumu   = (coeffs[5] + coeffs[4])/2. / normalization;
+   const auto C9p_bsmumu  = (coeffs[7] + coeffs[6])/2. / normalization;
+
+   const auto C10_bsmumu  = (coeffs[5] - coeffs[4])/2. / normalization;
+   const auto C10p_bsmumu = (coeffs[7] - coeffs[6])/2. / normalization;
+
+   const auto CS_bsmumu   = (coeffs[3] + coeffs[2])/2. / normalization / mb;
+   const auto CSp_bsmumu  = (coeffs[1] + coeffs[0])/2. / normalization / mb;
+
+   const auto CP_bsmumu   = (coeffs[3] - coeffs[2])/2. / normalization / mb;
+   const auto CPp_bsmumu  = (coeffs[1] - coeffs[0])/2. / normalization / mb;
+
+   // TODO(all): Where are sigma-sigma operators?
+
+   nlohmann::json j;
+   j["eft"] = "WET";
+   j["basis"] = "flavio";
+   j["scale"] = dynamic_cast<@ModelName@_mass_eigenstates const&>(context.model).get_scale();
+   j["values"] = {
+      {"C9_bsmumu",   {{"Re", Re(C9_bsmumu)},   {"Im", Im(C9_bsmumu)}}},
+      {"C9p_bsmumu",  {{"Re", Re(C9p_bsmumu)},  {"Im", Im(C9p_bsmumu)}}},
+      {"C10_bsmumu",  {{"Re", Re(C10_bsmumu)},  {"Im", Im(C10_bsmumu)}}},
+      {"C10p_bsmumu", {{"Re", Re(C10p_bsmumu)}, {"Im", Im(C10p_bsmumu)}}},
+      {"CS_bsmumu",   {{"Re", Re(CS_bsmumu)},   {"Im", Im(CS_bsmumu)}}},
+      {"CSp_bsmumu",  {{"Re", Re(CSp_bsmumu)},  {"Im", Im(CSp_bsmumu)}}},
+      {"CP_bsmumu",   {{"Re", Re(CP_bsmumu)},   {"Im", Im(CP_bsmumu)}}},
+      {"CPp_bsmumu",  {{"Re", Re(CPp_bsmumu)},  {"Im", Im(CPp_bsmumu)}}},
+   };
+   std::ofstream wc_json("WET_bsmumu.json");
+   wc_json << std::setw(4) << j << std::endl;
+   wc_json.close();
+
+}
+
+} // anonymous namespace
+
+using namespace @ModelName@_cxx_diagrams;
+using namespace @ModelName@_FFV_form_factors;
+
+namespace @namespace@ {
+
+typedef std::valarray<std::complex<double>>  (*ffv_function)
+   (int, int, const @ModelName@_mass_eigenstates&, bool);
+
+typedef std::array<std::complex<double>, 10> (*npf_function)
```

Comment:
It seems the magic number `10` appears multiple times in this file. Maybe we could give it a name like `constexpr NUMBER_OF_<dontknow>{10};`. Same for the magic number `4` in the line below.